CHANGES.rst

```diff
@@ -1,3 +1,12 @@
+v5.2.0 (2018-04-05)
+-------------------
+
+fixes:
+
+- Message: Allow partial message size
+- Slack: include the option to set Proxy when connecting using SlackClient
+- Slack: fix member joined channel event handler
+
```

Comment:
I don't think this was meant to be committed. If you could rebase this out so only the changes to `slack.py` and `setup.py` remain I'll be happy to merge. :smile: